src/HotChocolate/Core/src/Types.Scalars/ScalarResources.resx

```diff
@@ -319,4 +319,14 @@
   <data name="IPv4Type_ValidationPattern" xml:space="preserve">
     <value>(^(?:(?:(?:0?0?[0-9]|0?[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}(?:0?0?[0-9]|0?[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(?:\/(?:[0-9]|[1-2][0-9]|3[0-2]))?)$)</value>
   </data>
+  <data name="RegularExpressionType_Description" xml:space="preserve">
+    <value>The Regular Expression scalar type represents a String Type following a regular expression pattern/format, as defined
```

Comment:
The Regular Expression scalar type represents textual data, represented as UTF‐8 character sequences following a pattern defined as a regular expression.